pkgs/development/python-modules/catboost/default.nix

```diff
@@ -0,0 +1,56 @@
+{ buildPythonPackage, fetchFromGitHub, fetchpatch, lib, pythonOlder
+, clang_7, python2
```

Comment:
I must have been drunk when merging this. We can't use python2 or python3 as inputs in pythonPackages.

Reply:
Upstream scripts invoked during the build of native modules are python2-only. I don't think that upstream developers are willing to upgrade them because CatBoost still supports Python 2 installation.